geode-core/src/main/java/org/apache/geode/internal/statistics/platform/LinuxProcFsStatistics.java

```diff
@@ -334,6 +335,57 @@ private static void getMemInfo(int[] ints) {
     }
   }
 
+  /*
+   * TcpExt:=0 SyncookiesSent=1
+   * ListenOverflows=20 ListenDrops=21
+   */
+  private static void getNetStatStats(long[] longs, int[] ints) {
+    InputStreamReader isr;
+    BufferedReader br = null;
+    try {
+      isr = new InputStreamReader(new FileInputStream("/proc/net/netstat"));
+      br = new BufferedReader(isr);
+      String line;
+      do {
+        br.readLine(); // header
+        line = br.readLine();
+      } while (line != null && !line.startsWith("TcpExt:"));
+
+      st.setString(line);
+      st.skipTokens(1);
+      long tcpSyncookiesSent = st.nextTokenAsLong();
+      long tcpSyncookiesRecv = st.nextTokenAsLong();
+      st.skipTokens(17);
+      long tcpListenOverflows = st.nextTokenAsLong();
+      long tcpListenDrops = st.nextTokenAsLong();
+
+      longs[LinuxSystemStats.tcpExtSynCookiesRecvLONG] = tcpSyncookiesRecv;
+      longs[LinuxSystemStats.tcpExtSynCookiesSentLONG] = tcpSyncookiesSent;
+      longs[LinuxSystemStats.tcpExtListenDropsLONG] = tcpListenDrops;
+      longs[LinuxSystemStats.tcpExtListenOverflowsLONG] = tcpListenOverflows;
+
+      br.close();
```

Comment:
I'm going to change this to cache the somaxconn setting.  There's not much point in reading the setting over and over again since it will only change if someone edits the file.